package.json

```diff
@@ -195,11 +192,9 @@
     "mocha-multi-reporters": "^1.5.1",
     "npm-run-all": "^4.1.5",
     "opn": "^6.0.0",
-    "optimist": "0.3.5",
     "p-all": "^1.0.0",
```

Comment:
p-all is also used in exactly one place which could easily be replaced with a queue :)